### PR TITLE
SLING-10324 : ServiceUserMapper documentation is incomplete 

### PR DESCRIPTION
**Pull request by anchela**
(+ referring to API deprecation with SLING-10321)

### CODE REVIEW

File: src/main/jbake/content/documentation/the-sling-engine/service-authentication.md [hunk 1]
```diff
@@ -100,15 +100,23 @@ described above consists of three parts:
...
+user IDs ([SLING-6939](https://issues.apache.org/jira/browse/SLING-6963)). This mapping is configurable
 such that system administrators are in full control of assigning users to services.
 
 The `ServiceUserMapper` defines the following API:
 
     #!java
+    Iterable<String> getServicePrincipalNames(Bundle bundle, String subServiceName);
+     
+The alternative API (getting service user ID as shown below) has been deprecated for security reasons and will be removed
+in future releases. See [SLING-6939](https://issues.apache.org/jira/browse/SLING-6963) for details.
```
Comment (kwin) [inline src/main/jbake/content/documentation/the-sling-engine/service-authentication.md:113]: 6939 and link points to 6963. Which ticket tracks the deprecation? https://issues.apache.org/jira/browse/SLING-6963 is only about the introduction of the new API.

Comment (anchela) [reply src/main/jbake/content/documentation/the-sling-engine/service-authentication.md:113]: oh.... thanks for spotting... that's obviously an oversight. the deprecation is covered with SLING-10321. i will fix it.

Comment (kwin) [reply src/main/jbake/content/documentation/the-sling-engine/service-authentication.md]: Still the other link title says `SLING-6939` while the link actually points to `https://issues.apache.org/jira/browse/SLING-6963`

Comment (anchela) [reply src/main/jbake/content/documentation/the-sling-engine/service-authentication.md]: @kwin , well... that's an old mistake :-) but i can fix it as well...

**Review by kwin — APPROVED** [review]

**Comment by anchela** [comment]
@karlpauls , unless you have any concerns or objections, i would go ahead and merge the doc changes.

**Review by karlpauls — APPROVED** [review]
LGTM - thank you!

**Pull request merged**